R/use_cmakelists.R

```diff
@@ -0,0 +1,85 @@
+#'
+#'
+#' @export
+use_cmakelists <- function(project = NULL) {
+  writeLines(
+    c(
+      "cmake_minimum_required(VERSION 3.0.0)",
+      paste0("project(", project, " VERSION 0.1.0)"),
+      "set(CMAKE_EXPORT_COMPILE_COMMANDS ON)",
+      "include(CTest)",
+      "enable_testing()",
+      "",
+      "add_subdirectory(src)",
+      "set(CPACK_PROJECT_NAME ${PROJECT_NAME})",
+      "set(CPACK_PROJECT_VERSION ${PROJECT_VERSION})",
+      "include(CPack)",
+      "set(CMAKE_EXPORT_COMPILE_COMMANDS ON)"
+    ), "CMakeLists.txt"
+  )
+
+  if (!fs::dir_exists("src")) fs::dir_create("src")
```

Comment:
Don't even need to check:

```suggestion
  fs::dir_create("src")
```